src/selectedtests/app/app.py

```diff
@@ -8,14 +10,13 @@
     project_test_mappings_controller,
 )
 from selectedtests.datasource.mongo_wrapper import MongoWrapper
+from selectedtests.helpers import get_evg_api, get_mongo_wrapper
 
 
-def create_app(mongo_wrapper: MongoWrapper, evg_api: EvergreenApi) -> FastAPI:
+def create_app() -> FastAPI:
```

Comment:
These can be injected in tests using the container, rather than needing to be passed (a common theme for this DI stuff).